dist/constants.json

```diff
@@ -8919,1235 +8919,1235 @@
       },
       {
         "value": "663GA",
-        "label": "BELLEVUE VA CLINIC"
+        "label": "Bellevue VA Clinic"
       },
       {
         "value": "663GB",
-        "label": "BREMERTON VA CLINIC"
+        "label": "Silverdale VA Clinic"
       },
       {
         "value": "663GD",
-        "label": "SOUTH SOUND VA CLINIC"
+        "label": "South Sound VA Clinic"
       },
       {
         "value": "663GC",
-        "label": "MOUNT VERNON VA CLINIC"
+        "label": "Mount Vernon VA Clinic"
       },
       {
         "value": "663GE",
-        "label": "PORT ANGELES VA CBOC"
+        "label": "North Olympic Peninsula VA Clinic"
       },
       {
         "value": "663",
-        "label": "SEATTLE VA MEDICAL CENTER"
+        "label": "Seattle VA Medical Center"
       },
       {
         "value": "668",
-        "label": "MANN-GRANDSTAFF VAMC"
+        "label": "Mann-Grandstaff Department of Veterans Affairs Medical Center"
       },
       {
         "value": "687",
-        "label": "JONATHAN M. WAINWRIGHT VAMC"
+        "label": "Jonathan M. Wainwright Memorial VA Medical Center"
       }
     ],
     "RI": [
       {
         "value": "650GD",
-        "label": "MIDDLETOWN CBOC"
+        "label": "Middletown VA Clinic"
       },
       {
         "value": "650QA",
-        "label": "EAGLE SQUARE VA CLINIC"
+        "label": "Eagle Square VA Clinic"
       },
       {
         "value": "650QB",
-        "label": "EAGLE STREET VA CLINIC"
+        "label": "Eagle Street VA Clinic"
       },
       {
         "value": "650",
-        "label": "PROVIDENCE VAMC"
+        "label": "Providence VA Medical Center"
       }
     ],
     "DC": [
       {
         "value": "688GB",
-        "label": "SOUTHEAST WASHINGTON VA CLINIC"
+        "label": "Southeast Washington VA Clinic"
       },
       {
         "value": "688QA",
-        "label": "FRANKLIN STREET VA CLINIC"
+        "label": "Franklin Street VA Clinic"
       },
       {
         "value": "688",
-        "label": "WASHINGTON VA MEDICAL CENTER"
+        "label": "Washington VA Medical Center"
       }
     ],
     "CT": [
       {
         "value": "689A4",
-        "label": "NEWINGTON"
+        "label": "Newington VA Clinic"
       },
       {
         "value": "689GA",
-        "label": "WATERBURY"
+        "label": "Waterbury VA Clinic"
       },
       {
         "value": "689GB",
-        "label": "STAMFORD VA PRIMARY CARE CTR"
+        "label": "Stamford VA Clinic"
       },
       {
         "value": "689GC",
-        "label": "WINDHAM CBOC"
+        "label": "Willimantic VA Clinic"
       },
       {
         "value": "689GD",
-        "label": "WINSTED"
+        "label": "Winsted VA Clinic"
       },
       {
         "value": "689GE",
-        "label": "DANBURY"
+        "label": "Danbury VA Clinic"
       },
       {
         "value": "689HC",
-        "label": "NEW LONDON"
+        "label": "John J. McGuirk Department of Veterans Affairs Outpatient Clinic"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.

Reply:
Confirmed the name based on the city this facility is located in: New London, CT (https://www.va.gov/find-locations/facility/vha_689HC).